.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -81,10 +81,10 @@ jobs:
       - name: Build and Push Docker Image
         run: |
           cd netmanager/
-          docker build --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} .
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }}
-          docker tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-platform-frontend:latest
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-platform-frontend:latest
+          docker build --tag gcr.io/airqo-250220/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} .
+          docker push gcr.io/airqo-250220/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }}
+          docker tag gcr.io/airqo-250220/airqo-platform-frontend:${{ needs.image-tag.outputs.build_id }} gcr.io/airqo-250220/airqo-platform-frontend:latest
```

Comment:
Same here, is the system failing to pick up our ENVS? Why are we not able to use our secrets? @123MwanjeMike

Reply:
Yes @Baalmart we are no longer adding them as secrets given it's not secret information. One needs the credentials files to login and access those images plus they are already specified in all our k8s files.

@danielogen can help confirm this.